bindings/python/dnet_client

```diff
@@ -130,6 +130,7 @@ def process_backend():
         'enable': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.enable_backend),
         'disable': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.disable_backend),
         'defrag': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.start_defrag),
+        'compact': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.start_compact),
```

Comment:
There is a new question now: what are 'heavily fragmented' blobs? (those where size of all removed records is more than defrag_percentage, correct?)